install/kubernetes/helm/meshery/README.md

```diff
@@ -1,98 +1,74 @@
-# Meshery
+# meshery
+
+![Version: 2.1.0](https://img.shields.io/badge/Version-2.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: stable-latest](https://img.shields.io/badge/AppVersion-stable--latest-informational?style=flat-square)
+
+Meshery chart for deploying Meshery and Meshery's adapters.
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| Layer5 Authors | community@layer5.io |  |
+| aisuko | urakiny@gmail.com |  |
+| leecalcote | lee.calcote@layer5.io |  |
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+|  | meshery-consul | 2.0.0 |
+|  | meshery-istio | 2.0.0 |
+|  | meshery-kuma | 1.0.0 |
+|  | meshery-linkerd | 2.0.0 |
+|  | meshery-nsm | 1.0.0 |
+|  | meshery-osm | 1.0.0 |
```

Comment:
Thank you for this, @Aisuko. This looks nice. 

The Traefik Mesh and NGINX SM adapters are ready for incorporation.

Reply:
No problems. And `meshSync`

Reply:
Oh, right! // @dhruv0000